dm/openapi/spec/dm.yaml

```diff
@@ -1617,3 +1765,35 @@ components:
       required:
         - "total"
         - "data"
+
+    BatchImportTaskConfigRequest:
+      type: object
+      properties:
+        overwrite:
+          type: boolean
+          default: false
+          description: whether to overwrite task config template
+      required:
+        - "overwrite"
+    BatchImportTaskConfigResponse:
+      type: object
+      properties:
+        success_task_list:
+          type: array
+          items:
+            type: string
```

Comment:
How about return the success_task_config_list, not only the task_name

Reply:
seems webui  don't need to see taskconfigs, actually it only care about the failed msg 😅
you can see the Product prototype drawings [here](https://docs.google.com/document/u/2/d/1OT70m8rVdMILETZLz0hdpIftP2Mo_jvH66mY9VAtj-E/edit?pli=1&disco=AAAAQ_UCKv4)